src/slack/slack.js

```diff
@@ -885,12 +967,17 @@ export class Slack {
       fallback: ' ', // To avoid warning message in logs
     }
 
-    const orderConfirmation = await this.boltApp.client.chat.postMessage({
-      channel: user,
-      attachments: [orderAttachment],
-      as_user: true,
-      text: ' ',
-    })
+    let orderConfirmation
+    try {
+      orderConfirmation = await this.boltApp.client.chat.postMessage({
```

Comment:
let's do this similar to `getComments` - in try block, have both `postMessage` and `return`, in catch block, log error and re-throw - the data returned would not be useful, so better to indicate error upwards